### PR TITLE
feat: popup swap page

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MF-6123 MF-5736 MF-6125 MF-6126

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "dictionaries": ["typescript", "node", "html", "css", "fonts"],
     "ignorePaths": [
         "./packages/icons/countries/**",
+        "./packages/mask/popups/pages/swap/utils/**",
```
Comment (Jack-Works) [inline cspell.json]:
```suggestion
        "./packages/mask/popups/pages/swap/webgl/**",
```

File: packages/mask/popups/Popup.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ import { queryClient } from '@masknet/shared-base-ui'
 
 const Wallet = lazy(() => import(/* webpackPreload: true */ './pages/Wallet/index.js'))
 const Personas = lazy(() => import(/* webpackMode: 'eager' */ './pages/Personas/index.js'))
-const SwapPage = lazy(() => import('./pages/Swap/index.js'))
+// const SwapPage = lazy(() => import('../swap/pages/Swap/index.js'))
```
Comment (guanbinrui) [inline packages/mask/popups/Popup.tsx]: Remove commted code and according files if will not anymore.

File: packages/mask/swap/components/AccountManager.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+    const providerDescriptor = useProviderDescriptor(pluginID)
+    const networkDescriptor = useNetworkDescriptor(pluginID, chainId)
+
+    const handleDisconnect = useCallback(async () => {
+        await EVMWeb3.disconnect({
+            providerType,
+        })
+        onClose()
+        return
```
Comment (guanbinrui) [inline packages/mask/swap/components/AccountManager.tsx:77]:
```suggestion
```

File: packages/mask/swap/pages/Swap/SwapBackground.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import { useEffect, useRef, useState } from 'react'
```
Comment (guanbinrui) [inline packages/mask/swap/pages/Swap/SwapBackground.tsx]: Dup?

File: packages/web3-providers/src/Web3/Base/storage.ts [hunk 5]
```diff
@@ -57,6 +57,7 @@ export function providerStorage<ChainId extends number, ProviderType extends str
     defaultChainId: ChainId,
     defaultProviderType: ProviderType,
 ) {
+    console.log(getSiteType())
```
Comment (guanbinrui) [inline packages/web3-providers/src/Web3/Base/storage.ts]:
```suggestion
```

File: packages/mask/.webpack/config.ts [hunk 6]
```diff
@@ -395,6 +398,7 @@ async function addHTMLEntry({
     gun?: boolean
     perf: boolean
 }) {
+    console.log(options.filename)
```
Comment (guanbinrui) [inline packages/mask/.webpack/config.ts:401]:
```suggestion
```

File: packages/web3-providers/src/Web3/Base/storage.ts (hunk 5 above)
Comment (guanbinrui) [inline packages/web3-providers/src/Web3/Base/storage.ts]:
```suggestion
```

**Review by guanbinrui — APPROVED** [review]

File: packages/mask/swap/Swap.tsx [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+import { queryClient } from '@masknet/shared-base-ui'
+import { queryPersistOptions } from '../shared-ui/index.js'
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
+import { PageUIProvider } from '@masknet/shared'
+
+const SwapRoutes = memo(function SwapRoutes() {
+    return (
+        <Routes>
+            <Route
+                path="/"
```
Comment (Jack-Works) [inline packages/mask/swap/Swap.tsx:17]: if you only have one path, why do you need a router?

**Pull request merged**